DOCUMENTATION/content/documentation/index.md

```diff
@@ -569,38 +569,29 @@ docker-compose up -d php-worker
 ## Run Laravel Scheduler
 
 Laradock provides 2 ways to run Laravel Scheduler
-1 - Using cron in workspace container. Most of the time, when you start Laradock, it'll automatically start workspace container with cron inside, along with setting to run `schedule:run` command every minute.
-
-2 - Using Supervisord in php-worker to run `schedule:run`. This way is suggested when you don't want to start workspace in production environment.
-<br>
-a) Comment out cron setting in workspace container, file `workspace/crontab/laradock`
-
-```bash
-# * * * * * laradock /usr/bin/php /var/www/artisan schedule:run >> /dev/null 2>&1
-```
-<br>
-b) Create supervisor configuration file (for ex., named `laravel-scheduler.conf`) for Laravel Scheduler in `php-worker/supervisord.d/` by simply copy from `laravel-scheduler.conf.example`
-<br>
-c) Start php-worker container
-
-```bash
-docker-compose up -d php-worker
-```
-
-
-
-
-
-
+1. Using cron in workspace container. 
```

Comment:
I wonder why you change bullet style from ```-``` to ```.``` from here? There are lots of ```-``` above and it makes document inconsistent.
Is it possible to have screenshot of 2 bullet styles, so that we can see and compare which one is better?